packages/core/src/Stores/ui-store.js

```diff
@@ -495,6 +496,11 @@ export default class UIStore extends BaseStore {
         this.is_close_uk_account_modal_visible = is_open;
     }
 
+    get is_desktop() {
+        // TODO: remove tablet once there is a design for the specific size.
+        return this.is_tablet || this.screen_width > MAX_TABLET_WIDTH;
+    }
+
```

Comment:
Logically it refers to tablets, however we're naming it `is_desktop`
It'll be a bit confusing as we already have flag for desktop